gtkm/auth/database/database.py

```diff
@@ -0,0 +1,17 @@
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+
+SQLALCHEMY_DATABASE_URL = "sqlite:///./gtkm/auth/database/user_db.db"
+
+engine = create_engine(SQLALCHEMY_DATABASE_URL, connect_args={"check_same_thread": False})
+SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
```

Comment:
z dużej pisze się tylko nazwy klas, a nie instancje

Reply:
W [dokumentacji sql alchemy](https://docs.sqlalchemy.org/en/14/orm/session_api.html) znalazłem, że w przykładowym kodzie obiekt Session (analogiczny do tutejszego SessionLocal) jest nazwany z dużej litery

Reply:
Kierowałem się przykładami dostępnymi w dokumentacji. W dodatku to wywołanie tworzy niejako factory instancji klasy Session - np. w funkcji `get_db()` jest instrukcja `db = SessionLocal()`, która wykorzystuje obiekt zwrócony przez `sessionmaker`. Po zmianie nazwy na "zgodną ze sztuką" otrzymalibyśmy `db = local_session()`, mimo że tak naprawdę bardziej działa to jak konstruktor, niż zwykła funkcja. Tak czy siak - dyskusja chyba czysto akademicka, jak dla mnie mogą być obie wersje 😉

Reply:
Jesli tak jest w dokumentacji to niech będzie